src/Components/RecsListTable/index.js

```diff
@@ -34,6 +35,8 @@ import messages from '../../Messages';
 import {
   RECS_LIST_INITIAL_STATE,
   updateRecsListFilters as updateFilters,
+  updateRecsListSortIndex as SortRecsTableIndex,
```

Comment:
A nitpick: `SortRecsTableIndex` is a bid odd choice for the function name. I quite recommend to be consistent within the code and use 1) snake case, 2) verbs "update", "set" for such functions. E.g., consider
```suggestion
  updateRecsListSortIndex as updateSortIndex,
```